README.md

```diff
@@ -6,21 +6,44 @@ Currently supports classic & bendy butt feed types.
 
 ## API
 
-### `directMessageKey(x_dh_secret, x_dh_public, x_feed, y_dh_public, y_feed) => dmKey`
+### `directMessageKey(x_dh_secret, x_dh_public, x_feed_id, y_dh_public, y_feed_id) => dmKey`
+
+If you are _encrypting_ a DM, `x` is if your feed, and `y` is the person you are DM'ing.
+If you are _decryping_ a DM, `y` is if your feed, and `x` is the message author's.
 
-Assuming a direct message is being sent between feed "x" and feed "y", a shared DM key is derived where:
 - `x_dh_secret`, `x_dh_public` are feed x's diffie-hellman keys
-- `x_feed` is the feedId of x in BFE format
+- `x_feed_id` is the feedId of `x`
 - `y_dh_public` is feed y's diffie-hellman public key
-- `y_feed` is the feedId of y in BFE format
+- `y_feed_id` is the feedId of `y`
 
-All inputs are BFE encoded buffers.
+All inputs are [BFE encoded][BFE] buffers.
 The output `dmKey` is a plain buffer.
 
-### `directMessageKey.easy(myKeys) => sharedKey(feedId) => dmKey`
+
+### `directMessageKey.easy(myKeys) => makeKey(feedId) => dmKey`
 
 Convenience function which wraps `directMessageKey`
-Creates a diffie-hellman shared key between you and another feedId.
+Creates a diffie-hellman shared key between you and another `feedId`.
+
+
+### `poBoxKey(x_dh_secret, x_dh_public, x_id, y_dh_public, y_id) => pobKey`
+
+If you are _encrypting_ to a P.O. Box, then `x` is your feed, and `y` is the P.O. Box.
+If you are _decryping_ a message sent to a P.O. Box, then `x` is the P.O. Box, and `y` is the author.
```

Comment:
can we do message author here or did you leave that out on purpose?

Reply:
:+1: